tailwind.config.js

```diff
@@ -6,6 +6,16 @@ module.exports = {
   ],
   theme: {
     extend: {},
+    colors: {
```

Comment:
Why have to put it in colours, why not extend the colours with the original colours provided by tailwind theme so that we can use the preexisting ones

Reply:
okay so I should put the colors object inside the extend only am I right?